doc/classes/Node.xml

```diff
@@ -129,6 +129,7 @@
 				    child_node.get_parent().remove_child(child_node)
 				add_child(child_node)
 				[/codeblock]
+				[b]Note:[/b] If you want a child to be persisted to a [PackedScene], you must call [method set_owner] in addition to [method add_child]. This is typically relevant for [url=https://godot.readthedocs.io/en/latest/tutorials/misc/running_code_in_the_editor.html]tool scripts[/url] and [url=https://godot.readthedocs.io/en/latest/tutorials/plugins/editor/index.html]editor plugins[/url]. If [method add_child] is called without [method Object.set_owner], the newly added [Node] will not be visible in the scene tree, though it will be visible in the 2D/3D view.
```

Comment:
There's another occurrence of `[method set_owner]` which should be fixed. And I was wrong in my previous comment, `set_owner` is indeed a method of `Node` and not `Object`, but it's exposed via a property, so this should be `[member owner]`.

You can validate changes with `make -C doc rst`.